initial_setup/common.py

```diff
@@ -147,5 +147,9 @@ def get_quit_message():
 class LicensingCategory(SpokeCategory):
     displayOnHubGUI = "ProgressHub"
     displayOnHubTUI = "SummaryHub"
-    sortOrder = 100
-    title = N_("LICENSING")
+
+    def get_title():
```

Comment:
I think you need to specify `@staticmethod` again.

Reply:
Done. :)